Collecting cryptacular==1.5.5 (from -r D:/Windsor/WindsorThesis/Git repo/Thesis-Project-Python-Dependency-Conflict-Resolution/Dataset/gist/main_gist\requirements.7540117.txt (line 1))
  Downloading cryptacular-1.5.5.tar.gz (39 kB)
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
  Preparing metadata (pyproject.toml): started
  Preparing metadata (pyproject.toml): finished with status 'error'
  error: subprocess-exited-with-error
  
  Preparing metadata (pyproject.toml) did not run successfully.
  exit code: 1
  
  [22 lines of output]
  scons: Reading SConscript files ...
  ModuleNotFoundError: No module named 'imp':
    File "C:\Users\ASUS\AppData\Local\Temp\pip-install-wzwv7hrs\cryptacular_03cffafff0fd4574bac796c11ef02a8d\SConstruct", line 9:
      import enscons, enscons.cpyext
    File "C:\Users\ASUS\AppData\Local\Temp\pip-build-env-3dtpzco4\overlay\Lib\site-packages\enscons\cpyext.py", line 13:
      import imp
  Traceback (most recent call last):
    File "D:\Windsor\WindsorThesis\Git repo\Thesis-Project-Python-Dependency-Conflict-Resolution\pipTest\myenv\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 353, in <module>
      main()
    File "D:\Windsor\WindsorThesis\Git repo\Thesis-Project-Python-Dependency-Conflict-Resolution\pipTest\myenv\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 335, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "D:\Windsor\WindsorThesis\Git repo\Thesis-Project-Python-Dependency-Conflict-Resolution\pipTest\myenv\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 149, in prepare_metadata_for_build_wheel
      return hook(metadata_directory, config_settings)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "C:\Users\ASUS\AppData\Local\Temp\pip-build-env-3dtpzco4\overlay\Lib\site-packages\enscons\api.py", line 35, in prepare_metadata_for_build_wheel
      return _run("dist_info")
             ^^^^^^^^^^^^^^^^^
    File "C:\Users\ASUS\AppData\Local\Temp\pip-build-env-3dtpzco4\overlay\Lib\site-packages\enscons\api.py", line 29, in _run
      lookup = SCons.Node.arg2nodes_lookups[0](alias).sources[0]
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  AttributeError: 'NoneType' object has no attribute 'sources'
  [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

Encountered error while generating package metadata.

See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

[notice] A new release of pip is available: 23.2.1 -> 24.1.2
[notice] To update, run: D:\Windsor\WindsorThesis\Git repo\Thesis-Project-Python-Dependency-Conflict-Resolution\pipTest\myenv\Scripts\python.exe -m pip install --upgrade pip
